examples/analog_hamiltonian_simulation/03_Parallel_Jobs_on_Aquila.ipynb

```diff
@@ -0,0 +1,578 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    show_global_drive(drive)
</pre><p>Add <code>;</code> to the end prevent priting the string representation of the returned Axes object.</p><p> </p><p>(Same with the other plots below)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/205/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>